### ISSUE TITLE
flynn run bash returns 404 on unpushed

### ISSUE THREAD

**Issue by holtgrewe** [flynn/flynn#2776]
When running `flynn run bash` prints an uninformative 404 error on a project that has been created but no push has been done.

**Comment by lmars** [comment]
@holtgrewe when I create an app and then run `flynn run bash`, I get a slightly more informative message:

```
$ flynn create test
Created test

$ flynn -a test run bash
No app release, specify a release with -release
```

This is expected because the app has no release (i.e. I haven't pushed any code to create a release).

Can you perhaps include the commands you ran and the exact output that you saw?

**Comment by holtgrewe** [comment]
I think I also set some environment variables

```
$ flynn create flynnpoison
Created flynnpoison
$ flynn env set foo=bar
Created release dd891c37-bbed-425f-830c-17e8a3494750.
$ flynn run bash
POST https://controller.cubipaas.bihealth.org/apps/flynnpoison/jobs: httpclient: unexpected status 404
```

**Comment by lmars** [comment]
@holtgrewe ok I see the issue, and it is in fact more of a bug on the nightly release (it causes `flynn-host` to panic). Fix proposed in #2805.
